src/components/Drawer.tsx

```diff
@@ -7,68 +7,18 @@ import { DrawerItems } from "../helper/DrawerItems";
 import Typography from "@mui/material/Typography";
 import AccountTreeOutlinedIcon from "@mui/icons-material/AccountTreeOutlined";
 
-export const drawerWidth: number = 230;
-
-export function Drawer() {
-  const [open, setOpen] = React.useState(true);
-
-  const DrawerMod = styled(MuiDrawer, {
-    shouldForwardProp: (prop) => prop !== "open",
-  })(({ theme, open }) => ({
+export default function Drawer() {
+  const DrawerMod = styled(MuiDrawer)(({ theme }) => ({
     "& .MuiDrawer-paper": {
       position: "relative",
-      whiteSpace: "nowrap",
-      background: theme.palette.grey[100],
-      width: drawerWidth,
-      transition: theme.transitions.create("width", {
-        easing: theme.transitions.easing.sharp,
-        duration: theme.transitions.duration.enteringScreen,
-      }),
-      boxSizing: "border-box",
-      ...(!open && {
-        overflowX: "hidden",
-        transition: theme.transitions.create("width", {
-          easing: theme.transitions.easing.sharp,
-          duration: theme.transitions.duration.leavingScreen,
-        }),
-        width: theme.spacing(7),
-        [theme.breakpoints.up("sm")]: {
-          width: theme.spacing(9),
-        },
-      }),
+      background: theme.palette.background.default,
```

Comment:
The Background of the Drawer Component should be grey and the overall background white. If this is too much effort because of the theme we need to change it in the mockup aswell.